src/Microsoft.DotNet.Darc/src/Darc/Operations/GetDependencyGraphOperation.cs

```diff
@@ -193,20 +277,60 @@ private string GetGraphVizNodeName(DependencyGraphNode node)
             return GetSimpleRepoName(node.RepoUri).Replace("-", "") + node.Commit;
         }
 
-        private void LogGraphViz(DependencyGraph graph)
+        private async void LogGraphViz(DependencyGraph graph)
         {
-            Console.WriteLine("digraph repositoryGraph {");
-            Console.WriteLine("    node [shape=record]");
-            foreach (DependencyGraphNode node in graph.Nodes)
+            string directory = Path.GetDirectoryName(_options.GraphVizOutputFile);
+            if (!Directory.Exists(directory))
             {
-                Console.WriteLine($"    {GetGraphVizNodeName(node)}[label=\"{GetSimpleRepoName(node.RepoUri)}\\n{node.Commit.Substring(0, 5)}\"];");
-                foreach (DependencyGraphNode childNode in node.Children)
+                Directory.CreateDirectory(directory);
+            }
+
+            using (StreamWriter writer = new StreamWriter(_options.GraphVizOutputFile))
```

Comment:
>GraphVizOutputFile [](start = 67, length = 18)

What format is this file in... it seems to have a lot of weird escaped new lines in it. Is there a readme or a command line help that explains it?

Reply:
The format is dot, which has been around a long time. The escaped newlines are from the labels on the nodes.  Node names in the description have limited character sets.

Reply:
oh cool. Glad we are using something standard.  Could there be a comment or something in this method to reflect that? Or maybe name the method something to that effect.  I didn't even know that format existed.

Reply:
Will do